src/features/metadata-based-view/MetadataBasedItemList.js

```diff
@@ -58,30 +67,50 @@ class MetadataBasedItemList extends React.Component<Props, State> {
             }
 
             if (index === 1) {
```

Comment:
Should these indices now reference the new constants `FILE_ICON_COLUMN_INDEX` and `FILE_NAME_COLUMN_INDEX`?